src/common/clientserver.c

```diff
@@ -105,6 +105,30 @@ Com_VPrintf(int print_level, const char *fmt, va_list argptr)
 		Con_Print(msg);
 	#endif
 
+		if (!strcmp(YQ2OSTYPE, "Linux"))
+		{
+			/* change console color */
+			switch(msg[0])
+			{
+				// red
+				case 0x01:
+					Sys_ConsoleOutput("\033[40m\033[31;1m");
+					msgLen --;
+					memmove(msg, msg + 1, msgLen);
+					break;
+				// green
+				case 0x02:
+					Sys_ConsoleOutput("\033[40m\033[32;1m");
+					msgLen --;
+					memmove(msg, msg + 1, msgLen);
+					break;
+				// white
+				default:
+					Sys_ConsoleOutput("\033[40m\033[37;1m");
+					break;
+			}
+		}
+
```

Comment:
Where are these colors used anyways? :) Nevertheless, I would just leave this alone. Terminal output is hard if one starts to use special features. This misses a few things. At least:

* Linux isn't the only system with colored terminals.
* One cannot assume that a terminal supports colors. Even if most modern terminals do, some do not. Terminal combined with screen readers are often monochrome. Braille lines are terminals with next to no advanced features. And so on. This must be combined with termcap / terminfo checks to make sure, that the terminal supports colored output.
* The control sequences must not be emitted if the terminal isn't interactive. Otherwise they may clutter redirection into a file and things like that.

Reply:
I have only Linux, so I suppose it is useful for Linux only. 

I will move it to separate PR if i have some more technically correct solution.